src/main/scala/io/delta/tables/DeltaTable.scala

```diff
@@ -681,4 +678,38 @@ object DeltaTable {
     }
     isDeltaTable(sparkSession, identifier)
   }
+
+  /**
+   * :: Evolving ::
+   *
+   * Create an empty DeltaTable with the provided `schema` at the provided file `path`.
+   *
+   * @since 0.6.0
+   */
+  @Evolving
+  def createEmpty(sparkSession: SparkSession, schema: StructType, path: String): DeltaTable = {
+    val emptyDataFrame =
+      sparkSession.createDataFrame(sparkSession.sparkContext.emptyRDD[Row], schema)
+    emptyDataFrame.write.format("delta").save(path)
```

Comment:
I'll try the second approach and see if it works.

Reply:
I've put some thought into it and I'm thinking that instead of creating a new code path, it would be better to just optimize the existing write path to better handle this case.